README.md

```diff
@@ -12,6 +12,19 @@ itself.
 
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
+## Compatibility matrix
+
+|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|---------------------|----------------|----------------|----------------|----------------|
+| **Dashboard 1.4**   | ✓              | ✕              | ✕              | ✕              |
+| **Dashboard 1.5**   | ✕              | ✓              | ✕              | ✕              |
+| **Dashboard 1.6**   | ✕              | ✕              | ✓              | ?              |
+| **Dashboard HEAD**  | ✕              | ✕              | ?              | ✓              |
+
+- `✓` All the functionalities are supported with given version.
+- `?` Some of the functionalities might not be working.
```

Comment:
'x' known not to work. bugs will not be fixed
'?' maybe working. bugs will not be fixed